docs/develop/sdks/terra-js/coin-and-coins.md

```diff
@@ -21,6 +21,4 @@ const c = new Coin('uluna', 123.3); // a DecCoin
 const d = new Coin('uluna', '123.3'); // a DecCoin
 ```
 
-If it is ambiguous,
-
-Although it is convenient to represent the numbers through JavaScript's native `Number` format, you should 
+Although it is convenient to represent the numbers through JavaScript's native `Number` format, you should refrain from doing so.
```

Comment:
@evanorti Not sure what the intended idea was hear. Let me know if what I interpolated seems incorrect.

Reply:
@ericHgorski Not sure what was meant either, someone else wrote this part, but your edit is clear.